packages/react-native-gesture-handler/src/v3/types/ConfigTypes.ts

```diff
@@ -25,7 +25,9 @@ export type GestureCallbacks<THandlerData> = {
     event: GestureStateChangeEvent<THandlerData>,
     didSucceed: boolean
   ) => void;
-  onUpdate?: (event: GestureUpdateEvent<THandlerData>) => void | AnimatedEvent;
+  onUpdate?:
+    | ((event: GestureUpdateEvent<THandlerData>) => void)
+    | AnimatedEvent;
```

Comment:
Instead of a function or an `AnimatedEvent`, it was a function that was returning void or `AnimatedEvent`.

This was irritating, as it was the reason types in `InterceptingGestureDetector` were correct.